### DISCUSSION TITLE
posts/deal-with-ini-file/

### DISCUSSION THREAD

**Discussion by giscus** [BYJRK/byjrk.github.io#16] [Announcements]
# posts/deal-with-ini-file/

INI 文件是一种相当古老的配置文件格式，但如今仍然被广泛使用。本文将会介绍几种在 C# 中读写 INI 文件的方法，并探讨它们的效率和适用场景。

https://blog.coldwind.top/posts/deal-with-ini-file/

**Comment by MorningCigarette** [comment]
沙发

**Comment by fusrodahs** [comment]
大佬试一下config.net呢

**Reply by BYJRK** [reply]
我还没用过，研究一下
